Please fix [lint failure](https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/7620635330?check_suite_focus=true#step:7:5246):

```
# github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver [github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver.test]
Error: .\kafka_receiver_test.go:680:18: cannot use t (type *testConsumerGroup) as type require.TestingT in argument to require.NoError:
	*testConsumerGroup does not implement require.TestingT (missing Errorf method)
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver [build failed]
FAIL
```

--- a/receiver/kafkareceiver/kafka_receiver_test.go
+++ b/receiver/kafkareceiver/kafka_receiver_test.go
@@ -12,7 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// nolint:errcheck
 package kafkareceiver
 
 import (
@@ -678,7 +677,7 @@ var _ sarama.ConsumerGroup = (*testConsumerGroup)(nil)
 
 func (t *testConsumerGroup) Consume(_ context.Context, _ []string, handler sarama.ConsumerGroupHandler) error {
 	t.once.Do(func() {
-		handler.Setup(testConsumerGroupSession{})
+		_ = handler.Setup(testConsumerGroupSession{})
 	})
 	return t.err
 }

